controllers/api/v1alpha1/runworkload_types.go

```diff
@@ -45,9 +45,9 @@ type RunWorkloadSpec struct {
 	MemoryMiB int64 `json:"memoryMiB"`
 	// +kubebuilder:validation:Minimum:=1
 	// +kubebuilder:validation:Required
-	DiskMiB int64 `json:"diskMiB"`
-	// +kubebuilder:validation:Format:=uint8
-	CPUWeight uint8 `json:"cpuWeight"`
+	DiskMiB              int64 `json:"diskMiB"`
+	CPURequestMillicores int64 `json:"cpuRequestMillicores"`
+	CPULimitMillicores   int64 `json:"cpuLimitMillicores"`
```

Comment:
Did we agree to take out limits in the end? Does it mean we only keep a single `CPUMillicores` field?

Reply:
> since the override will come in a different story.

So why not introduce the limit property as part of that story? Having an unused placeholder property
 (effectively dead code) is not great as every time someone is doing whatever with workloads, they would have to rediscover that limits are never used - pretty sure that I would have forgotten about that by the end of the day :)

Reply:
That's a good point. I'll remove the limits and rename requests to CPUMillicores for now and the following story can add the new limit param and necessary functionality.